samples/src/Microsoft.Azure.IIoT.App/src/Validation/DiscovererInfoValidator.cs

```diff
@@ -86,5 +91,13 @@ public class DiscovererInfoValidator : AbstractValidator<DiscovererInfoRequested
 
             return false;
         }
+
+        private bool BeAValidDiscoveryUrl(List<string> value) {
+            if (value != null) {
+                return !(value.Contains(null) || value.Contains(string.Empty));
+            }
+
+            return true;    
```

Comment:
When value == null, it will return true. Is this expected?

Reply:
null means empty list, no discovery Urls set